corehq/apps/reports/templates/reports/filters/bootstrap5/multi_option.html

```diff
@@ -2,25 +2,36 @@
 {% load hq_shared_tags %}
 {% block filter_content %}
   {% if not endpoint %}
-    <select class="{{ css_class }} form-control report-filter-multi-option"
-            multiple="true"
-            data-bind="options: select_params, optionsText: 'text', optionsValue: 'val', selectedOptions: current_selection"
-            data-options="{% html_attr select.options %}"
-            data-selected="{% html_attr select.selected %}"
-            placeholder="{{ select.placeholder }}"
-            name="{{ slug }}"></select>
+    <select
+      class="{{ css_class }} form-control report-filter-multi-option"
+      name="{{ slug }}"
+      multiple="true"
+      placeholder="{{ select.placeholder }}"
+      data-bind="
+        ptions: select_params,
+        optionsText: 'text',
+        optionsValue: 'val',
+        selectedOptions: current_selection
+      "
+      data-options="{% html_attr select.options %}"
+      data-selected="{% html_attr select.selected %}"
+    ></select>
   {% else %}
-    <select data-endpoint="{% html_attr endpoint %}"
-            data-options="{% html_attr select.options %}"
-            data-selected="{% html_attr select.selected %}"
-            class="{{ css_class }} form-control report-filter-multi-option"
-            placeholder="{{ select.placeholder }}"
-            name="{{ slug }}"></select>
+    <select
+      class="{{ css_class }} form-control report-filter-multi-option"
```

Comment:
```suggestion
      class="{{ css_class }} form-select report-filter-multi-option"
```

Reply:
thanks. likely wasn't seen because of the select2 styling that eventually dominates the select
fixed in https://github.com/dimagi/commcare-hq/pull/35704/commits/6fd62f71fcbd2fc56936b57f3388471d4498df88